website/docs/r/linux_function_app.html.markdown

```diff
@@ -97,6 +97,8 @@ The following arguments are supported:
 
 * `client_certificate_exclusion_paths` - (Optional) Paths to exclude when using client certificates, separated by ;
 
+~> **Note:**   TLS 1.3 and HTTP 2.0 don't support TLS renegotiation. These protocols will not work if your app is configured with client certificate settings that use TLS renegotiation. Either disable ClientCert, or set ClientCertMode to Application or Required (without exclusion paths).
```

Comment:
It's not immediately clear what `ClientCert` or `ClientCertMode` are in Terraform terms. We should have those Terraform variable names in the note instead